client/src/analysis/types.ts

```diff
@@ -81,6 +83,18 @@ export const AnalysisMetadata: { [s: string]: AnalysisMetadataItem } = {
         desc: "Pick a single frame",
         title: "Pick frame",
     },
+
+    [AnalysisTypes.APPLY_FFT_MASK]: {
+        desc: "Pick a single frame",
+        title: "FFT",
+        component: FFTAnalysis
+    },
+    
+    [AnalysisTypes.FFTSUM_FRAMES]: {
+        desc: "Create a sum of all detector frames",
+        title: "Sum all frames",
```

Comment:
fft of sum of all frames, no? even though this one is currently not displayed...